packages/grafana-ui/src/utils/logs.ts

```diff
@@ -23,6 +23,15 @@ export function getLogLevel(line: string): LogLevel {
   return LogLevel.unknown;
 }
 
+export function getLogLevelFromKey(key: string): LogLevel {
+  const level = (LogLevel as any)[key];
```

Comment:
Interesting line of code!

Reply:
Isn't it :smile: Didn't knew how to solve the typecheck error so looked at the existing getLogLevel above. What would be the proper solution, `Object.Keys(LogLevel)[key]`?